VMs/metrics/README.md

```diff
@@ -137,6 +153,63 @@ $ ./7_delete_clone.sh ubuntu16.04
 # Otherwise, now you have a master VM set up you are ready to configure Jenkins.
 ```
 
+## Configuring the user on the host
+
+A user needs to be chosen on the host system that will execute the VMs. That user
+requires some configuration. The following documents an installtion with the user
+`jenkins`. The host system and user require some setup to enable use of the scripts.
+
+### Add golang `bin` directories to path
+
+The user must have both the GOLANG bin and their GOPATH bin directories in their
+path. This can be effected by adding the following to their .profile:
+
+```bash
+# set PATH so it includes golang
+if [ -d "/usr/local/go/bin" ] ; then
+	PATH="/usr/local/go/bin:$PATH"
+fi
+
+export GOPATH=${HOME}/go
+
+# set PATH so it includes local go bin
+if [ -d "${GOPATH}/bin" ] ; then
+	PATH="${GOPATH}/bin:$PATH"
+fi
+```
+
+### Add user to VM groups
+
+In order for the user to launch VMs via `virsh`, they need to be a member of some
+extra groups:
+
+```bash
+$ adduser jenkins libvirt
+$ adduser jenkins libvirt-qemu
+```
+
+### Installing the `agent.jar`
+
+The scripts require the Jenkins `agent.jar` file to be located in the `${HOME}/bin`
+directory of the user. This file can be obtained from your Jenkins master:
+
+```bash
+$ mkdir -p ${HOME}/bin
+$ cd ${HOME}/bin
+$ curl  http://jenkins.katacontainers.io/jnlpJars/agent.jar --output agent.jar
```

Comment:
Nit: double space after `curl`. Also, for safety and succinctness, you could make this:

```
$ curl -LO http://jenkins.katacontainers.io/jnlpJars/agent.jar
```